src/ui/atoms/ButtonYesNo/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import type { IProps } from './types/IProps';
+import styles from './styles.module.scss';
+
+function Button({ children, variant = 'primary' }: IProps) {
+  const colorFont = styles[`button-label--${variant}`];
```

Comment:
You should use another approach to set up your variants, check this example to fix your pr . https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10